processes/processes_test.go

```diff
@@ -329,9 +448,6 @@ func (mp Proc) validateValue(param string, value uint64) bool {
 	case "ps_cputime_system":
 		stime, _ := strconv.ParseUint(string(mp.Stat[14]), 10, 64)
 		refValue = stime * 10000
-	case "ps_count":
```

Comment:
This case was used in previous implementation and it is not possible to check it here after refactor, as it varies depending on process instances count. Previously there was just 1 process instance per process mocked and value "1" hardcoded, now there are 2 processes, first with 1 instance mocked, second with 2 instances. For `ps_count` I created 2 separate tests, please check "check process instance count with..." conveys.